src/views/evaluations/evaluations.js

```diff
@@ -47,8 +46,8 @@ function EvaluationsTableHeader(options) {
         });
 
         const headingsEl = `
```

Comment:
I'd argue that this can just be `const headingsEl = headings.join('');`. Would you agree?